<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNodeWidget</class>
 <widget class="QWidget" name="AddNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 3px solid rgb(23, 7, 0);
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
	stop:0 rgb(255,204,150),
	stop:0.167 rgb(255,204,150),
	stop:0.1670001 rgb(255,250,240)
	stop:1 rgb(255, 250, 240));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);
border: none;
font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>汇聚</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>111</y>
      <width>258</width>
      <height>168</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,0);
border-color: rgba(255, 255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>此处显示图像</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	image: url(:/nodewidget/addTerminal.png);
	border-radius: 5px;
	border: 0px;
	background-color: rgb(250, 235, 200);
}

QPushButton:hover {
	background-color: rgb(220,165,130);
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid rgb(220,165,130);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="delButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    image: url(:/nodewidget/delTerminal.png);
	border-radius: 5px;
	border: 0px;
	background-color: rgb(255,235,200);
}

QPushButton:hover {
	background-color: rgb(220,165,130);
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid rgb(220,165,130);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-radius: 10px;
image: url(:/nodewidget/ADD.png);
background: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border: none;
background-color: rgba(255, 255, 255, 0);
font: 10pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>105</y>
      <width>270</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px dashed rgb(100, 100, 100);
border-radius: 10px;
background-color:rgb(230, 230, 230);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 10px;
	border-image: url(:/nodewidget/close.png);
	background: rgb(255,255,255);
}

QPushButton:hover {
	background-color: rgb(220,220,220);
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid rgb(220,220,220);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>addButton</zorder>
   <zorder>delButton</zorder>
   <zorder>label_3</zorder>
   <zorder>checkBox</zorder>
   <zorder>closeButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
